server/server.go

```diff
@@ -182,6 +183,13 @@ func apiRoutes() *mux.Router {
 	r.HandleFunc(config.Webroot+"api/v1/chaos", middleWareFunc(apiv1.GetChaos)).Methods("GET")
 	r.HandleFunc(config.Webroot+"api/v1/chaos", middleWareFunc(apiv1.SetChaos)).Methods("PUT")
 
+	// Prometheus metrics (if enabled and using existing server)
+	if prometheus.GetMode() == "integrated" {
+		r.HandleFunc(config.Webroot+"metrics", middleWareFunc(func(w http.ResponseWriter, r *http.Request) {
+			prometheus.GetHandler().ServeHTTP(w, r)
```

Comment:
This appears to be causing issues when it comes to gzip compression (double-zipped - try accessing in a browser). I suspect `prometheus.GetHandler().ServeHTTP()` is adding gzip, however so is the `middleWareFunc()`